Microsoft.Toolkit.Uwp.SampleApp/SamplePages/RangeSelector/RangeSelectorCode.bind

```diff
@@ -6,32 +6,33 @@
     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
     xmlns:converters="using:Microsoft.Toolkit.Uwp.UI.Converters"
     mc:Ignorable="d">
-    <Page.Resources>
-        <converters:StringFormatConverter x:Key="StringFormatConverter"/>
-    </Page.Resources>
+  <Page.Resources>
+    <converters:StringFormatConverter x:Key="StringFormatConverter"/>
+  </Page.Resources>
 
-    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-        <Grid VerticalAlignment="Center" 
-		      HorizontalAlignment="Center">
-            <Grid.ColumnDefinitions>
-                <ColumnDefinition Width="50"/>
-                <ColumnDefinition Width="200"/>
-                <ColumnDefinition Width="50"/>
-            </Grid.ColumnDefinitions>
-            <TextBlock Grid.Column="0" 
-                       HorizontalAlignment="Left"
-                       VerticalAlignment="Center"
-                       Foreground="Black"
-                       Text="{Binding RangeMin, ElementName=RangeSelectorControl, Converter={StaticResource StringFormatConverter}, ConverterParameter='{}{0:0.##}'}" />
-            <controls:RangeSelector x:Name="RangeSelectorControl"
-                                    Grid.Column="1"
-                                    Minimum="@[Minimum:Slider:0:0-100]@" 
-                                    Maximum="@[Maximum:Slider:100:0-100]@" />
-            <TextBlock Grid.Column="2" 
-                       HorizontalAlignment="Right"
-                       VerticalAlignment="Center"
-                       Foreground="Black"
-                       Text="{Binding RangeMax, ElementName=RangeSelectorControl, Converter={StaticResource StringFormatConverter}, ConverterParameter='{}{0:0.##}'}" />
-        </Grid>
+  <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Grid VerticalAlignment="Center"
+      HorizontalAlignment="Center">
+      <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="50"/>
+        <ColumnDefinition Width="200"/>
+        <ColumnDefinition Width="50"/>
+      </Grid.ColumnDefinitions>
+      <TextBlock Grid.Column="0"
+                 HorizontalAlignment="Left"
+                 VerticalAlignment="Center"
+                 Foreground="Black"
+                 Text="{Binding RangeMin, ElementName=RangeSelectorControl, Converter={StaticResource StringFormatConverter}, ConverterParameter='{}{0:0.##}'}" />
+      <controls:RangeSelector x:Name="RangeSelectorControl"
+                              Grid.Column="1"                              
+                              Minimum="@[Minimum:Slider:0:0-100]@"
+                              Maximum="@[Maximum:Slider:100:0-100]@"
+                              StepFrequency="@[StepFrequency:String:1]@"/>
```

Comment:
Can you make this a DoubleSlider so that they can just slide to change the step value and not guess what to type in as a string?

E.g.
`@[StepFrequency:DoubleSlider:1.0:0.25-10.0]@`

Reply:
yes this will be a good thing to add, it will make more sense.